tools/update_kernel_commits.py

```diff
@@ -47,15 +47,15 @@ def get_short_arch_flavor(branch_name):
     _, _, arch, _, flavor = branch_name.split("-", 4)
     return f"{arch}-{flavor}"
 
-
-def fetch_latest_commits_from_github(user_token):
+def fetch_latest_commits_from_github(user_token, verbose=False):
     """
-    Generate the kernel-commits.mk-new file with commit information from GitHub branches.
-
     This function retrieves branch information from a GitHub repository and
     returns the commit hashes for each branch in a set.
```

Comment:
Description must be updated as well, you are not retrieving protected branches....